schema/testdata.go

```diff
@@ -329,8 +329,7 @@ func GenTestData(table *Table, opts GenTestDataOptions) []arrow.Record {
 			if nullRow && !c.NotNull && !c.PrimaryKey &&
 				c.Name != CqSourceNameColumn.Name &&
 				c.Name != CqSyncTimeColumn.Name &&
-				c.Name != CqIDColumn.Name &&
-				c.Name != CqParentIDColumn.Name {
```

Comment:
This wasn't a bug as such, but I don't see a reason to prevent CQParentIDColumn from being set to null, as in many cases it will be null

Reply:
How a parentIDColumn can be null?

Reply:
For all top-level tables